### DISCUSSION TITLE
Проблема с базой данных

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#471] [Неопределенное]
Добрый день! Помогите разобраться. У нас по непонятной причине легла база. Ребутнули сервер и в логах нашли что какой то процесс потушил ее (красным на скрине подчеркнул).

Также заметили, что логи забиты сообщениями (оранжевым выделил). В чем может быть причина?

![111](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/80567813/28c24d1d-9770-4674-ae09-c13c5581b127)

**Comment by ttm-alexeypolunin** [comment]
И что пишут интернеты?

**Reply by a1ext0** [reply]
Смотрели, пишут, что это команда приостанавливает процесс извне. Но в логе не указан процесс, который это сделал. Поэтому подозреваем тотум в отправлении команды на терминейт.

**Comment by poluninatm** [comment]
Да, скорее всего тотум. Завтра полезу разбираться.
А до этих записей в логе у вас сообщений об ошибках не было?

Команда закрывает процесс - помнится, была добавлена для какого-то редкого случая лет
сколько-то назад. В убиении сервера бд ранее замечена не была🤷‍♀️

**Reply by Spyker086** [reply]
У нас почему то весь лог забит сообщениями, как на скрине (оранжевым выделено)

**Reply by poluninatm** [reply]
Посмотрела. Это сообщение мы отправляем базе при каждом откате транзакции - то есть при каждый раз при перезапуске транзакций (если одновременно несколько пользователей меняют таблицу), при критичных ексепшнах - например, из кодов-действий и тд. Массово они у вас в бд могли появиться при включении bugFinder'а - его мы активно тестировали и тестовая база у нас не чихнула.

База у вас так и ложится или это был единичный эпизод?

Место еще поииследую - оно ключевое и задействуется часто.

**Reply by poluninatm** [reply]
Из Доков: 

"Чтобы завершить отдельный сеанс, не прерывая работу других сеансов, воспользуйтесь функцией pg_terminate_backend() (см. [Таблицу 9.78](https://postgrespro.ru/docs/postgresql/10/functions-admin#FUNCTIONS-ADMIN-SIGNAL-TABLE))" - команда не должна глушить сервер БД

Предполагаю, что сервер был погашен кем-то другим или каким-то другим процессом
